core/src/main/scala/cats/Composed.scala

```diff
@@ -176,6 +176,18 @@ private[cats] trait ComposedContravariantCovariant[F[_], G[_]] extends Contravar
     F.contramap(fga)(gb => G.map(gb)(f))
 }
 
+private[cats] trait ComposedApplicativeDecidable[F[_], G[_]]
+    extends Decidable[λ[α => F[G[α]]]]
+    with ComposedApplicativeContravariantMonoidal[F, G] { outer =>
+  def F: Applicative[F]
+  def G: Decidable[G]
+
+  def sum[A, B](fa: F[G[A]], fb: F[G[B]]): F[G[Either[A, B]]] =
+    F.map2(fa, fb)(G.sum)
+
+  override val zero: F[G[Nothing]] = F.pure(G.zero)
```

Comment:
I think I may have steered you wrong and this may be triggering the val init order bug. Try changing to `lazy val` or `def` and see if it fixes:

```
   ==> X cats.tests.NestedSuite.initializationError0.001s java.lang.NullPointerException: Cannot invoke "cats.Decidable.zero()" because the return value of "cats.data.NestedDecidable.FG()" is null
      at cats.data.NestedDecidable.$init$(Nested.scala:411)
      at cats.data.NestedInstances1$$anon$10.<init>(Nested.scala:135)
      at cats.data.NestedInstances1.catsDataDecidableForApplicativeForNested(Nested.scala:135)
      at cats.tests.NestedSuite.<init>(NestedSuite.scala:144)
```

Reply:
Nice, thank you.